discogs_client/models.py

```diff
@@ -291,6 +291,9 @@ def _invalidate(self):
 
     def _load_pagination_info(self):
         data = self.client._get(self._url_for_page(1))
+        self._pages[1] = [
+                self._transform(item) for item in data[self._list_key]
+            ]
```

Comment:
Would you mind unindenting these last 2 lines? We indent by 4 spaces and generally like to keep the ending parenthesis/bracket/etc. aligned with where it opens. i.e.,
```
self._pages[1] = [
    self._transform(item) for item in data[self._list_key]
]
```